.unreleased/pr_7463

```diff
@@ -0,0 +1 @@
+Implements: #7463: Support SET NOT NULL on compressed hypertables
```

Comment:
I guess this is the changlog entry I meant comment on.

It is misleading because we only support SET NOT NULL on hypercore TAM tables or non-compressed tables. Also, since this is a change that applies only to Hypercore TAM, it is not necessary to have a changelog entry given that we haven't released Hypercore TAM yet and it already has a changlog entry.

Reply:
It does apply to tables that have compression parameter set, which is what I meant when I said "compressed hypertable". I agree that it can give false hopes, but I am open to suggestions.

I'll remove the changelog entry for now.